### ISSUE TITLE
Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

### ISSUE THREAD

**Issue by RilDev** [NightCatSama/vue-slider-component#601]
#### The Problem

With Node 16, the new Node LTS, I get a warning everytime I start my server.

```
: Using / for division outside of calc() is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div($size, 2) or calc($size / 2)

More info and automated migrator: https://sass-lang.com/d/slash-div

  ╷
7 │     border-width: $size / 2;
  │                   ^^^^^^^^^
  ╵
    node_modules/vue-slider-component/lib/styles/_triangle.scss 7:19  triangle()
    node_modules/vue-slider-component/lib/styles/_triangle.scss 31:7  arrow()
    node_modules/vue-slider-component/lib/theme/default.scss 108:7    @import
    src/components/Filters/Partials/FilterPrice.vue 419:9             root stylesheet
```

#### The Environment

 - OS & Version: Linux 20.0.4
 - Vue version: v2.6.12 (using Nuxt v2.15.8)
 - Component Version: v3.2.15

**Comment by kinoli** [comment]
I'm seeing the same thing. Would love a fix for this.

**Comment by ngenerio** [comment]
I think this has been [fixed already](https://github.com/NightCatSama/vue-slider-component/blob/master/lib/styles/_triangle.scss#L7) but hasn't been released yet

@NightCatSama , could you please create a new release for this fix. Thank you!

**Comment by lowbits** [comment]
push, please make a release. Waiting now 3 months for it. 🙏

**Comment by pandalion** [comment]
I am still getting warnings in this package too, will this be updated? thank you.

**Comment by peterpacket** [comment]
I wouldn't mind a release either!

**Comment by rbecheras** [comment]
Up, 3 more months have passed :-) 

Please just release a new patch to fix this

**Comment by NightCatSama** [comment]
@rbecheras In the latest version, it should have been replaced by `* 0.5`

**Comment by rbecheras** [comment]
Ah yes, but it's the v4 beta... 

Ok, I understand it's not critical to push a patch to the v3.

**Comment by NightCatSama** [comment]
Fixed in version 3.2.23.

**Comment by rbecheras** [comment]
Thanks!
